liblangutil/ErrorReporter.h

```diff
@@ -107,6 +108,11 @@ class ErrorReporter
 		SecondarySourceLocation const& _secondaryLocation,
 		std::string const& _description = std::string());
 
+	void fatalError(Error::Type _type,
```

Comment:
wrong indentation / newlines.

Reply:
I used the same style as the functions above and below this one. 
I'll fix those as well then.